packages/office-addin-telemetry/src/telemetryJsonData.ts

```diff
@@ -0,0 +1,67 @@
+
+import * as chalk from "chalk";
+import * as fs from "fs";
+import * as os from "os";
+import * as path from "path";
+/**
+ * Allows developer to create prompts and responses in other applications before object creation
+ * @param groupName Event name sent to telemetry structure
+ * @param telemetryEnabled Whether user agreed to data collection
+ * @returns Boolean of whether the program should prompt
+ */
+export function promptForTelemetry(groupName: string, jsonFilePath): boolean {
+    try {
+        const jsonData: any = readTelemetryJsonData(jsonFilePath);
+        if (jsonData) {
+            return !groupNameExists(jsonData, groupName);
+        }
+        return true;
+    } catch (err) {
+        console.log(chalk.default.red(err));
+    }
+}
+
+/**
+ * Reads data from the telemetry json config file
+ * @param jsonFilePath Path to the json config file
+ * @returns Parsed object from json file if it exists
+ */
+export function readTelemetryJsonData(jsonFilePath: string): any {
```

Comment:
Let's set a default jsonFilePath here as well like you are doing for the writeJson methods